### ISSUE TITLE
NMP -v

### ISSUE THREAD

**Issue by terenceholland** [nodejs/help#1794]
* **Win 10 Enterprise**:
* **Install**:

I just installed Node.js from https://nodejs.org/en/download/
The installation seemed to go without issue but when I open a terminal in Visual Studio Code and type >npm -v I get 2,7.4 returned.   This is a much lower version than I am expecting and lower than the minimum required to use Angular 2 (the only reason for me installing node)

I followed the same steps on my home setup and, whilst I cant remember the exact version, it was high enough to run angular.  This points to there being somme issue with my work pc setup.  Could you pls advise on what I should be looking for to resolve this issue.

Thanks

**Comment by Hakerh400** [comment]
Run `npm i -g npm` to update npm. Latest stable version is 6.8.0

**Comment by terenceholland** [comment]
Tried this.  Still on 2.7.4 after.

Windows PowerShell
Copyright (C) Microsoft Corporation. All rights reserved.

PS C:\Users\myusername> npm i -g npm
npm WARN package.json path-is-inside@1.0.2 No README data
npm WARN package.json sorted-object@2.0.1 No README data
npm WARN package.json punycode@1.4.1 punycode is also the name of a node core module.
npm WARN package.json string_decoder@1.2.0 string_decoder is also the name of a node core module.
C:\Users\myusername\AppData\Roaming\npm\npm -> C:\Users\myusername\AppData\Roaming\npm\node_modules\npm\bin\npm-cli.js
C:\Users\myusername\AppData\Roaming\npm\npx -> C:\Users\myusername\AppData\Roaming\npm\node_modules\npm\bin\npx-cli.js
npm@6.8.0 C:\Users\myusername\AppData\Roaming\npm\node_modules\npm
PS C:\Users\myusername> npm -v
2.7.4
PS C:\Users\myusername>

**Comment by Hakerh400** [comment]
Please post the output of

```
whene npm
"%appdata%\npm\npm.cmd" -v
```

**Comment by terenceholland** [comment]
Not sure I understand.  If you are asking where npm.cmd is the it is in 

C:\Users\myusername\AppData\Roaming\npm

**Comment by Hakerh400** [comment]
I mean paste that literally in your terminal, press enter and provide the full output. The problem is, as it currently seems, that your Visual Studio Code uses different location for NPM (internal installation) instead of the global one. But the output of the above command would reveal more info.

**Comment by terenceholland** [comment]
Windows PowerShell
Copyright (C) Microsoft Corporation. All rights reserved.

PS C:\Users\myusername> where npm
PS C:\Users\myusername> "%appdata%\npm\npm.cmd" -v
At line:1 char:25
+ "%appdata%\npm\npm.cmd" -v
+                         ~~
Unexpected token '-v' in expression or statement.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : UnexpectedToken
 
PS C:\Users\myusername>

**Comment by terenceholland** [comment]
sorry - let me try again

**Comment by terenceholland** [comment]
Microsoft Windows [Version 10.0.16299.967]
(c) 2017 Microsoft Corporation. All rights reserved.

C:\Users\myusername>where npm
C:\Program Files (x86)\nodejs\npm
C:\Program Files (x86)\nodejs\npm.cmd
C:\Program Files\nodejs\npm
C:\Program Files\nodejs\npm.cmd
C:\Users\myusername\AppData\Roaming\npm\npm
C:\Users\myusername\AppData\Roaming\npm\npm.cmd

C:\Users\myusername>"%appdata%\npm\npm.cmd" -v

C:\Users\myusername>

**Comment by Hakerh400** [comment]
You have multiple Node.js installations. Uninstall one of them or remove it from PATH environment variable.

**Comment by terenceholland** [comment]
Thank you - this is now resolved
